.github/PULL_REQUEST_TEMPLATE/new_component_template.md

```diff
@@ -0,0 +1,20 @@
+## Describe your changes (can list changeset entries if preferable)
+
+
+## Author Checklist (complete before requesting a review)
+- [ ] I have performed a self-review of my code
+- [ ] I have added thorough tests where applicable (unit / component / visual)
+- [ ] If visual tests are required for the component, I have created a project in Percy
+- [ ] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
+- [ ] I have reviewed visual test updates properly before approving
+- [ ] If changes will affect consumers of the package, I have created a changeset entry.
+- [ ] If a changeset file has been created, I have used the `/snapit` functionality to test my changes in a consuming application
+
```

Comment:
suggestion: "I have created a new label in Github for this component and added it to the labeller"

Reply:
great shout!

Reply:
Very good idea, maybe a link to the file too, for good measure?
https://github.com/justeattakeaway/pie/blob/main/.github/project-labeler.yml

Reply:
All done!